### PR TITLE
Create booking show, routes and form

### PR DESCRIPTION
**Pull request by mhzehl**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   # for the Creative bootstrap.. info https://github.com/MrPowers/frontend-generators
 
 root to: 'creatives#index'
-
+resources :bookings, only: [:new, :create]
```
Comment (a-bolz) [inline config/routes.rb:6]: hier moet toch ook een index, show, edit, update en delete?

**Review by a-bolz — COMMENTED** [review]
zie comment in de routes. afgezien daarvan :rocket:

File: app/views/shared/_booking-widget.html.erb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  <div>
+    <%= f.label("Check-in") %>
+    <%= f.date_field(:start_date) %>
+  </div>
+
+  <div>
+    <%= f.label("Check-out") %>
+    <%= f.date_field(:end_date) %>
+  </div>
+
```
Comment (timkuilman) [inline app/views/shared/_booking-widget.html.erb:11]: You can do ranges here, with an iteration --> (1..15) A lot shorter!

**Review by timkuilman — COMMENTED** [review]
You can shorten the form, by using a range (1..15)

File: app/views/shared/_booking-widget.html.erb (hunk 2 above)
Comment (mhzehl) [reply app/views/shared/_booking-widget.html.erb:11]: @timkuilman I figured, but didn't know how... Can you show me please...

**Pull request merged**